### PR TITLE
Rematch optimising

### PR DESCRIPTION
**Pull request by qifeng-bai**

### CODE REVIEW

File: grails-app/conf/application.yml
Comment (hamzajaved-csiro) [inline grails-app/conf/application.yml:1]: I think we should remove these changes as it was for our local envs

File: grails-app/conf/logback.xml
Comment (hamzajaved-csiro) [inline grails-app/conf/logback.xml:1]: Same for this

File: grails-app/controllers/au/org/ala/specieslist/WebServiceController.groovy [hunk 1]
```diff
@@ -32,6 +32,7 @@ import io.swagger.v3.oas.annotations.parameters.RequestBody
 import io.swagger.v3.oas.annotations.responses.ApiResponse
 import io.swagger.v3.oas.annotations.security.SecurityRequirement
 import org.apache.http.HttpStatus
+import grails.web.mime.MimeType
```
Comment (hamzajaved-csiro) [inline grails-app/controllers/au/org/ala/specieslist/WebServiceController.groovy:35]: Unused import

File: grails-app/domain/au/org/ala/specieslist/MatchedSpecies.groovy [hunk 2]
```diff
@@ -31,8 +31,6 @@ class MatchedSpecies {
     String genus
     Date lastUpdated
 
-    static belongsTo = [speciesListItem:SpeciesListItem]
```
Comment (hamzajaved-csiro) [inline grails-app/domain/au/org/ala/specieslist/MatchedSpecies.groovy:34]: Should we keep this relationship?

Comment (qifeng-bai) [reply grails-app/domain/au/org/ala/specieslist/MatchedSpecies.groovy:34]: We need to remove it.  'BelongsTo' will reverse the order of connection -> creates a species_list_item_id in matchedSpecies table.  It slows down the reading speed of species_list_item

Comment (hamzajaved-csiro) [reply grails-app/domain/au/org/ala/specieslist/MatchedSpecies.groovy:34]: Ah sorry, just saw the change for SpeciesListItem

**Review by hamzajaved-csiro — APPROVED** [review]

**Pull request closed without merging**